ngraph/frontend/paddlepaddle/src/op/exp.cpp

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <ngraph/opsets/opset8.hpp>
+#include <node_context.hpp>
+
+namespace ngraph {
+namespace frontend {
+namespace pdpd {
+namespace op {
+NamedOutputs exp(const NodeContext& node) {
+    auto data = node.get_ng_input("X");
+    PDPD_OP_VALIDATION_CHECK(node, data.get_element_type() != ov::element::f64, "exp not support fp64 input type !");
```

Comment:
This should not be done in the frontend. If fp64 is not supported in inference_engine that is not a problem of a frontend.  If you observe test fails you can create ticket for this problem to be fixed on IE side.

Reply:
have removed the check. it's a common limitation of IE, so maybe we add this limitation description to  'paddle op conversion document' later?

Reply:
agree